File: src/main/java/um/haberes/report/model/haberes/core/FacultadDto.java
package um.haberes.report.model.haberes.core;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import um.haberes.report.util.Jsonifyable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FacultadDto implements Jsonifyable {

    private Integer facultadId;

    private String nombre = "";

    private String reducido = "";

    private String server = "";

    private String backendServer = "";

    private int backendPort = 0;

    private String dbName = "";

    private String dsn = "";

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/FacultadDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import um.haberes.report.util.Jsonifyable

data class FacultadDto(

    var facultadId: Int? = null,
    var nombre: String = "",
    var reducido: String = "",
    var server: String = "",
    var backendServer: String = "",
    var backendPort: Int = 0,
    var dbName: String = "",
    var dsn: String = ""

) : Jsonifyable
